### DISCUSSION TITLE
Mac Mouse Fix needs to be re-enabled when switching users

### DISCUSSION THREAD

**Discussion by rbongni** [noah-nuebling/mac-mouse-fix#478] [Questions]
I have two user accounts on my laptop. Each time I switch accounts, I need to open Mac Mouse Fix to re enable Mac Mouse fix to get it to work again (the 'Enable Mac Mouse Fix' checkbox is checked but I need to uncheck and re check it). Anyone else having this issue? Is there there a fix or workaround?

**Comment by oryandunn** [comment]
I also have this issue.  I just got my first Mac, and setup accounts for both me and my wife.  I had Mac Mouse Fix on my account, but when I switched to her account, scrolling didn't work.  I then enabled Mac Mouse Fix on her account, which got her account going, but when I switched back to my account, I had to disable and re-enable MacMouseFix on my account.  Awesome app, but this needs to be fixed.  I don't know if the service can be run at a system level so all users can benefit, perhaps with their own specific settings.

**Comment by muzzakus** [comment]
Man, I'd instantly recommend this app to everyone far and wide if this were prioritised.  Using a desktop Mac with family users it makes it a no go till fixed. Was so happy using this till the flaw became evident.

Is there some interim fix potential, e.g. a startup script restarting the app or something?

**Comment by gitstuffedm8** [comment]
Same here, would be cool to know a fix or if dev can fix it on the app. I have an automator workflow that turns off/on macmousefix when logging in users but the issue also happens when just switching user profiles. Annoying because i have different profiles for work.
